### PR TITLE
Switching to using the new Sdk

### PR DESCRIPTION
**Pull request by bettinaheim**
We are disabling importing core packages (since some are built on this repo). 
Currently, the projects will spawn a warning because we explicitly reference a package that the Sdk imports. This is temporary, since I am currently still lacking the whole set of packages.

### CODE REVIEW

File: updateQDKVersion.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ else
 fi
 
 : ${ver:="$NUGET_VERSION"}
-: ${pkgs:="Microsoft.Quantum.Development.Kit;Microsoft.Quantum.IQSharp.Core;Microsoft.Quantum.Simulators;Microsoft.Quantum.Compiler;Microsoft.Quantum.Canon;Microsoft.Quantum.Xunit;Microsoft.Quantum.Chemistry;Microsoft.Quantum.Research"}
+: ${pkgs:="Microsoft.Quantum.CsharpGeneration;Microsoft.Quantum.Runtime.Core;Microsoft.Quantum.QSharp.Core;Microsoft.Quantum.Development.Kit;Microsoft.Quantum.IQSharp.Core;Microsoft.Quantum.Simulators;Microsoft.Quantum.Compiler;Microsoft.Quantum.Xunit;Microsoft.Quantum.Chemistry;Microsoft.Quantum.Research"}
```
[the remark this answers is not in the document]
Comment (bettinaheim) [reply updateQDKVersion.sh]: Yes, we do. Will add that.

**Comment by bettinaheim** [comment]
> Looks good to me. I think the only real question I had was the name of the new .props file refactored out from the unit testing projects. In particular, we've generally used names like Dev.props to indicate properties used for local development, but those same properties are also used on CI builds in this case? Perhaps something like build/props/tests.props?

done.

**Comment by cgranade** [comment]
> done.

Awesome, thanks! I'll leave it to you to press the merge button whenever you're ready for this to go in, then. Thanks!

**Comment by bettinaheim** [comment]
> done.
> 
> Awesome, thanks! I'll leave it to you to press the merge button whenever you're ready for this to go in, then. Thanks!

I am all ready to go - we just need to actually keep the fix for the .NET Core 3.1 update, and then merge all 5 PRs together. Will do that shortly. Thank you for all the reviews, @cgranade!

**Pull request merged**